### ISSUE TITLE
add environment variable to disable accesslog in lighttpd

### ISSUE THREAD

**Issue by vladak** [pi-hole/docker-pi-hole#484]
This is a... 

- [x] Request for a new or modified feature
- [ ] Issue trying to run the docker image
- [ ] Issue trying to build / test / develop the docker image

## Description
I'd like to be able to disable access log in `lighttpd` process completely.

## Expected Behavior
No access logs are written by `lighttpd` if a tunable is enabled.

## Actual Behavior
Log entries are written to `/var/log/lighttpd/access.log`

## Possible Fix
Add a ENV variable to disable access log.

## Steps to Reproduce and debugging done
1. query the local server http://localhost/admin/api.php
2. see `/var/log/lighttpd/access.log` to be written into

## Context and extra information

In order to drive I/O writes down on my Raspberry Pi (in order to save the SD card it is running from) I tracked down which processes contribute the most and `lighttpd` running from Pi-hole was one of them. This is because I have a script that queries the `lighttpd` server every couple of seconds to retrieve Pi-hole statistics (so that it can be displayed on OLED display - https://learn.adafruit.com/pi-hole-ad-blocker-with-pi-zero-w/).

This request is similar to #337.

## Your Environment 

* Docker Host Operating System and OS Version: Raspbian GNU/Linux 10 (buster)
* Docker Version: 18.09.8
* Hardware architecture: ARMv7

**Comment by vladak** [comment]
Also, I tried to workaround it by placing `accesslog.file = "/dev/null"` to `/etc/lighttpd/external.conf` however this does not work. Plus, I encountered some issues when mapping `/etc/lighttpd/` as a volume in docker compose file because of the `conf-enabled` subdirectory.

Lastly, trying to workaround this by editing `lighttpd.conf` directly works however is not persistent across `docker-compose` `down/up` very much like described in #366.
